### PR TITLE
Drop keystore link in favor of standard Candlepin config

### PR DESCRIPTION
**Pull request by ehelms**

### CODE REVIEW

**Comment by ehelms** [comment]
Goes along with https://github.com/theforeman/puppet-certs/pull/225

**Comment by ehelms** [comment]
I can make that change from my testing. The change in certs is still needed as I want to move the keystore to the default Candlepin location and out of `/etc/pki/katello` intentionally to make each service and it's configuration more independent. Right now, you have to mount at least 4 different volumes to run Candlepin as a container. By centralizing these into `/etc/tomcat` and `/etc/candlepin` only two need to be mounted.

File: manifests/candlepin.pp [hunk 1]
```diff
@@ -22,6 +22,7 @@
     oauth_secret                 => $oauth_secret,
     ca_key                       => $::certs::ca_key,
     ca_cert                      => $::certs::ca_cert_stripped,
+		keystore_file								 => $certs::candlepin::keystore_file,
```
Comment (ekohl) [inline manifests/candlepin.pp]: Could you fix tabs vs spaces here?

**Comment by ehelms** [comment]
Updated and the change in puppet-certs has been merged

**Pull request merged**